### PR TITLE
[DOCS-6214] Identity Service 1.4

### PR DESCRIPTION
**Pull request by oliveralfresco2018**

### CODE REVIEW

File: identity-service/latest/support/index.md [hunk 1]
```diff
@@ -2,10 +2,11 @@
...
 ---
 
-The following are the supported platforms for the Identity Service version 1.3:
+The following are the supported platforms for the Identity Service version 1.4:
 
 | Version | Notes |
 | ------- | ----- |
+| Content Services 7.0.0 | Content Services supports the use of CMIS and authentication with the v1 REST APIs using the Identity Service. ADF and other modules are not currently supported for authentication. |
 | Content Services 6.2.2 | Content Services supports the use of CMIS and authentication with the v1 REST APIs using the Identity Service. ADF and other modules are not currently supported for authentication. |
 | Content Services 6.2.1 | Content Services supports the use of CMIS and authentication with the v1 REST APIs using the Identity Service. ADF and other modules are not currently supported for authentication. |
```
Comment (anxumalo) [inline identity-service/latest/support/index.md:11]: Is ACS 6.x supported?

**Review by anxumalo — CHANGES_REQUESTED** [review]
Hi @jamalkm, are you able to confirm if the details in `/latest/support/index.md` are correct? Many thanks

**Comment by stevenblair-al** [comment]
approved.

File: identity-service/latest/support/index.md (hunk 1 above)
Comment (anxumalo) [reply identity-service/latest/support/index.md:11]: Yes: confirmed in https://www.alfresco.com/services/subscription/technical-support/product-support-status

**Review by anxumalo — APPROVED** [review]

**Pull request merged**